components/Feature/VulnerabilitiesGrid/ResourceCard/index.js

```diff
@@ -32,9 +32,9 @@ const ResourceCard = ({
   const trimLength = (s, length) => s.length > length ? s.substring(0, length) + "..." : s
 
   const selfReferralElement = (selfReferral == 'No') ? 'Referral required' : 'Self referral'
-  const websiteElement = websites && websites.length > 0 &&  websites.map(website => (<a href={websites[0]} target="_blank" rel="noopener noreferrer">{websites[0]}</a>))
+  const websiteElement = websites && websites.length > 0 &&  websites.map(website => (<><a href={website} target="_blank" rel="noopener noreferrer">{website}</a><br/></>))
   const distributionElement =  tags.filter(t => HIDDEN_TAGS.includes(t)).join(", ")
-  const tagsElement = tags.filter(t => !HIDDEN_TAGS.includes(t)).map(item=> (<span key={"tags-"+item} className={`${css.tags} tag-element`}>{trimLength(item, 20)}</span>))
+  const tagsElement = tags.filter(t => !HIDDEN_TAGS.includes(t)).map(item=> (<span key={"tags-"+item} className={`${css.tags} tag-element ${css[`${item}-tag`]}`}>{trimLength(item, 20)}</span>))
```

Comment:
I'm a little bit confused by the `css.tags` bit, are you accessing actual css here or is css just some property name?

Reply:
also it looks like it was there already so we might want to rename that as part of separate pr

Reply:
The css is something that we're importing at the top `import css from './index.module.scss';`
And then we're accessing different scss parts from there.
I think this is the way Next makes you do styling

Reply:
https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css that's how nextjs handles component level css